app/http.php

```diff
@@ -31,6 +31,13 @@
     'validateCNAME' => false,
 ]);
 
+$register->set('cache', function () use ($register) { // Register cache connection
```

Comment:
If we keep this set callback, we can just replace the original one.

Reply:
The original one in `init.php` is used by the workers 👍🏻